src/diffusers/pipelines/auto_pipeline.py

```diff
@@ -66,20 +80,56 @@
         ("stable-diffusion", StableDiffusionInpaintPipeline),
         ("stable-diffusion-xl", StableDiffusionXLInpaintPipeline),
         ("if", IFInpaintingPipeline),
+        ("kandinsky", KandinskyInpaintCombinedPipeline),
+        ("kandinsky22", KandinskyV22InpaintCombinedPipeline),
+        ("stable-diffusion-controlnet", StableDiffusionControlNetInpaintPipeline),
+    ]
+)
+
+AUTO_TEXT2IMAGE_DECODER_PIPELINES_MAPPING = OrderedDict(
```

Comment:
In what context will these be used?

I'm thinking about potentially creating auto pipelines for SDXL, the combined one is clear, but where would the `base` and `refiner` steps go? Is `Decoder` a good name or too specific to these models?

Reply:
Yeah good point, it's not in the public API so we can definitely change the naming later. I'll make those mappings private for now to allow them to be changed after